contributors/guide/README.md

```diff
@@ -72,7 +72,7 @@ You get the idea - if you ever see something you think should be fixed, you shou
 ### Find a good first topic
 
 There are multiple repositories within the Kubernetes community and a full list of repositories can be found [here](https://github.com/kubernetes/).
-Each repository in the Kubernetes organization has beginner-friendly issues that provide a good first issue. For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [help wanted issues](https://go.k8s.io/help-wanted) that should not need deep knowledge of the system.
+Each repository in the Kubernetes organization has beginner-friendly issues that provide a good first issue. For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [good first issue](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) and [help wanted](https://go.k8s.io/help-wanted) labels for issues that should not need deep knowledge of the system.
```

Comment:
Hm. I wonder if we can differentiate a bit more here - make it clear that `good-first-issue` is meant to be an experience where you get guidance. Both labels are good for first-time contributors; it depends on each contributor's comfort zone.